### DISCUSSION TITLE
Running locally your own al-folio website | George C. de Araújo

### DISCUSSION THREAD

**Discussion by giscus** [george-gca/george-gca.github.io#34] [Announcements]
# Running locally your own al-folio website | George C. de Araújo

Step by step on how to run your own al-folio locally.

https://george-gca.github.io/blog/2022/running-local-al-folio/

**Comment by kasjfksj** [comment]
Hi, could you tell me specifically about giscus comment

**Reply by george-gca** [reply]
What do you mean?

**Comment by mtcq** [comment]
Dear George,
thank you very much for the support, It worked perfectly to me!  =)

Small observation, initially, I've tried to use a more recent version of Ruby, and I did
rbenv install 3.1.6
instead of
rbenv install 3.1.2
(and also rbenv local 3.1.6 instead of rbenv local 3.1.2).

For some reason I didn't understand, I couldn't proceed with "bundle install" using the 3.1.6 version.
However, when I've tried with the 3.1.2 version as you did and it worked perfectly.

All the best,
Marco

**Reply by george-gca** [reply]
Currently the template works with newer versions of Ruby.

**Comment by justuswill** [comment]
Needed to also run `sudo apt install -y build-essential libssl-dev zlib1g-dev libffi-dev libyaml-dev` before everything and `bundle init` before but worked, thanks!

**Reply by george-gca** [reply]
Thanks for pointing it out. I didn't need to run these commands, but probably because I had these already installed for some other application.
